storybook/stories/API/props/Legend.ts

```diff
@@ -16,6 +16,229 @@ export const legendType: StorybookArg = {
 // Caveat: If any prop is added here, it would falsely be add to the documentation of the component
 // where this group is used. If the group is to be extended, then only single props should be imported
 // by each component that does not use all of them.
-export const Legend: StorybookArgs = {
-  legendType,
+export const LegendProps: StorybookArgs = {
+  width: {
+    description: 'The width of legend.',
+    table: {
+      type: {
+        summary: 'Number',
+      },
+      category: 'General',
```

Comment:
we should probably use categories to prevent the other props from displaying up top such as dangerouslySetInnerHTML, etc.